config/environments/production.rb

```diff
@@ -54,7 +54,7 @@
 
   # Use a different cache store in production.
   # config.cache_store = :mem_cache_store
-  config.cache_store = :dalli_store, nil, { namespace: :government_frontend, compress: true }
+  config.cache_store = :mem_cache_store, nil, { namespace: :government_frontend, compress: true }
```

Comment:
Good spot.  Can the `dalli` gem also be removed from the Gemfile?

Reply:
Seems not — I had assumed so at first but it looks like `MemcacheStore` is also provided by the `dalli` gem. The other cases where we've had to fix this issue both retain the gem ([1](https://github.com/alphagov/local-links-manager/pull/751/files#diff-89cade48462044ee1b672dc5f4c3ec250fbd29effcd8932096a23c1283c6731fR107), [2](https://github.com/alphagov/collections/pull/2287/files#diff-d09ea66f8227784ff4393d88a19836f321c915ae10031d16c93d67e6283ab55fR5))